SPECS/mpi3mr-module.spec

```
@@ -0,0 +1,54 @@
+%define module_dir extra
+
+Summary: Driver for mpi3mr-module
```

I believe this is based on the Summary of the `igc-module` package. I didn't notice at the time that it isn't really meaningful.

A potential replacement:
```suggestion
Summary: Broadcom mpi3mr device drivers
```